docs/interactive-pause.md

```diff
@@ -40,10 +30,31 @@ vendor/bin/mftf run:manifest
 vendor/bin/mftf run:failed
 ```
 
-### MFTF Codecept Run Command
+### Use `Interactive Pause` During Test Development
+
+Here is a typical work flow for this use case:
+ 
+- Set `ENABLE_PAUSE` to `true` under `.env`
+- Add `<pause>` action in a test where you want to stop for investigation
```

Comment:
Add `<pause>` action in a test where you want to pause execution for debugging

Reply:
Replaced